voip_oca/models/mail_activity.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class VoipOcaActivity(models.Model):
+    _inherit = "mail.activity"
+
+    main_partner_id = fields.Many2one(
```

Comment:
I would suggest to make the module dependent on mail_activity_partner https://github.com/OCA/social/pull/1606 instead.

Reply:
Please don't. Base modules shouldn't depend on others not base one. I think the activity should be put to the user, not the partner.

Reply:
But we need the partner to show the partner that we are going to call. That's why we used this.

The problem with the module that you are suggesting is that we are adding the extra mail_board functionality that it is not required right now for us :thinking: